metatron-app/metatron-discovery
# #2680 improve engine monitoring ui

### Description
Improve engine monitoring ui

**Related Issue** : #2680 



### How Has This Been Tested?

Go to Engine monitoring
1. Select an interval to see if auto refresh is enabled.
2. Check the segment count.
3. Check the task, supervisor, and remote worker instant search on the Ingestion tab.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
